app/view/panel/application/StartViewController.js

```diff
@@ -2,18 +2,29 @@ Ext.define('MoMo.admin.view.panel.application.StartViewController', {
     extend: 'Ext.app.ViewController',
     alias: 'controller.momo-application-start-view',
 
+    /**
+     * 
+     */
     renderMap: function() {
         var me = this;
 
         me.setOlMap();
-        //me.addMapCrossHair();
         me.registerOnMapMoveEnd();
 
     },
 
+    /**
+     * 
+     */
     setOlMap: function() {
         var me = this,
             view = me.getView(),
+            viewModel = me.getViewModel(),
+            defaultData = viewModel.getData(),
+            zoom = defaultData.mapZoom,
+            x = defaultData.mapCenter.x,
+            y = defaultData.mapCenter.y,
+            projCode = defaultData.mapProjection,
             olMap;
 
         olMap = new ol.Map({
```

Comment:
At the moment the `ol.source.Stamen` sources will be used for the overview map. These layers are really nice, but unfortunately pretty empty for mongolian bbox. Maybe we could use the `OSM-WMS` layer from http://ows.terrestris.de/osm/service/ instead as this contains much more map elements?

It doesn't hurt for the moment, but could be a fast enhancement